sender.go

```diff
@@ -0,0 +1,20 @@
+package turn
+
+import (
+	"net"
+
+	"github.com/pion/stun"
+)
+
+// Sender is responsible for building a message and sending it to the given addr.
+type Sender func(conn net.PacketConn, addr net.Addr, attrs ...stun.Setter) error
```

Comment:
Oh that is really interesting! I bet that could be helpful if you want to insert a custom attr/inspect in every message. Nice idea.

Reply:
Thank you! Yeah, it will allow us to analyze in-depth every message.